lib/functions.php

```diff
@@ -90,6 +90,18 @@ function asyncCall(callable $callback, ...$args)
     {
         Promise\rethrow(call($callback, ...$args));
     }
+
+    /**
+     * Sleeps for the specified number of milliseconds.
+     *
+     * @param int $milliseconds
+     *
+     * @return Promise
+     */
+    function sleep(int $milliseconds): Promise
```

Comment:
Could you please change the name to `delay` to match `Delayed`?

Reply:
Done

Reply:
I meant changing the function name, not the return type.